server/main.lua

```diff
@@ -19,7 +19,7 @@ end
 ---@param player Player
 local function billPlayer(player)
 	player.Functions.RemoveMoney("bank", sharedConfig.checkInCost, "respawned-at-hospital")
-	exports.qbx_management:AddMoney("ambulance", sharedConfig.checkInCost)
+	exports['Renewed-Banking']:addAccountMoney("ambulance", sharedConfig.checkInCost)
```

Comment:
Should this be a config function instead so that it's more flexible to other banking scripts?

Reply:
Yeah that's probably the better way to do it. Let me rework it

Reply:
moved it to the config as requested.